src/main/java/InputData/newPlaylistInputData.java

```diff
@@ -0,0 +1,4 @@
+package InputData;
+
+public class newPlaylistInputData {
+}
```

Comment:
is this complete?

Reply:
> Some small comments, not required to merge but please look at them.

Also, why are the GitHub Auto-Grade tests failing? I'm not sure but we probably shouldn't merge until those pass.

Reply:
> is this complete?

Ah that was something I didn't get around to implementing yet, I forgot to check that off of the files to push

Reply:
You were trying to access private instance attributes without the getter, see my comment.

Reply:
> > Some small comments, not required to merge but please look at them.
> 
> Also, why are the GitHub Auto-Grade tests failing? I'm not sure but we probably shouldn't merge until those pass.

I'm unsure why that's happening as well